Tests/UnitTests/Core/Vertexing/LinearizedTrackFactoryTests.cpp

```diff
@@ -120,13 +126,15 @@ BOOST_AUTO_TEST_CASE(linearized_track_factory_test) {
   Vector4 vtxPos;
   double d0v{};
   double z0v{};
+  double t0v{};
   {
     double x = vXYDist(gen);
     double y = vXYDist(gen);
     double z = vZDist(gen);
-    vtxPos << x, y, z, 0;  // TODO: add time
+    t0v = vTDist(gen);
```

Comment:
```suggestion
    double t = vTDist(gen);
```
I think `t` should be part of `x,y,z` rather than `d0v,z0v`